File: 프로그래머스/lv2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    rows = len(arr1)
    cols = len(arr2[0])

    answer = [[0] * cols for _ in range(rows)]
    for i in range(rows):
        for j in range(cols):
            for k in range(len(arr2)):
                answer[i][j] += arr1[i][k] * arr2[k][j]
                
    return answer

File: 프로그래머스/lv2/12949. 행렬의 곱셈/행렬의 곱셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int rows = arr1.length;
        int cols = arr2[0].length;
        int[][] answer = new int[rows][cols];
        
        for(int r = 0; r < rows; r++){
            for(int c = 0; c < cols; c++){
                for(int k = 0; k < arr2.length; k++){
                    answer[r][c] += arr1[r][k] * arr2[k][c];
                }
            }
        }
        return answer;
    }
}